napari_organoid_counter/_widget.py

```diff
@@ -95,6 +99,10 @@ def __init__(self,
         self.stored_confidences = {}
         self.stored_diameters = {}
 
+        # Initialize multi_annotation_mode to False by default
+        self.multi_annotation_mode = False
```

Comment:
Having two boolean variables here is redundant, and makesb the code more error prone